src/VisualizationTests/HelixWatch3DViewModelTests.cs

```diff
@@ -1183,7 +1187,33 @@ public void CanTagGeometryWhenClickingArrayItemInWatchNode()
             tagGeometryWhenClickingItem(new[] { 0 }, 11, "Watch", 
                 n => n.ViewModel.NodeModel.InPorts[0].Connectors[0].Start.Owner);
         }
-        
+
+        [Test]
+        [Category("RegressionTests")]
+        public void AggregateRenderPackagesDoesntMutateRenderPackage()
+        {
+            // Regression test for DYN-5329:
+            // HelixWatch3DViewModel.AggregateRenderPackages event handler shouldn't mutate render packages.
+
+            OpenVisualizationTest("MultipleTextureMaps.dyn");
+
+            var output = ViewModel.CurrentSpace.NodeFromWorkspace<CreateList>(Guid.Parse("04fcc9b7f80b43c99923b0dac8930e77"));
+
+            output.RenderPackagesUpdated += (NodeModel nodeModel, RenderPackageCache renderPackages) => {
```

Comment:
@twastvedt please unsub this handler when the test is over, it's simpler than reasoning about if you need to or not with a memory profiler.

Reply:
Oh right. Yes, sorry.